scripts/db_migrator.py

```diff
@@ -419,6 +435,8 @@ def version_1_0_3(self):
         """
         log.log_info('Handling version_1_0_3')
 
+        self.migrate_feature_table()
```

Comment:
What is the reason for adding this here? Can this call be made as part of `common_migration_ops()`?

Reply:
1_0_2 has old FEATURE table
1_0_3 has new FEATURE table
There is no point to run this migrationg on > 1_0_3